### PR TITLE
Bump deps to support Rails 7

### PR DESCRIPTION
**Pull request by corasaurus-hex**
See the PR title. The one weirdness I noticed is that globalid 1.1.0 changes how globalid serializes to JSON which I imagine will cause us trouble in subtle ways.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -62,48 +57,48 @@ GEM
...
       rspec-core (>= 2, < 4, != 2.12.0)
     thor (1.2.1)
-    tzinfo (2.0.5)
+    tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
     zeitwerk (2.6.6)
 
 PLATFORMS
   arm64-darwin-21
-  ruby
```
Comment (Benjaminpjacobs) [inline Gemfile.lock:96]: Was this removal an artifact of the bundler update? I'm surprised we don't need this anymore.

File: spec/global_id_utils_spec.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@
       strs = ['gid://fish/NeonTetra/1', 'gid://fish/SiameseFighting/1']
       gids = strs.map { |str| GlobalID.new(str) }
 
-      expect(GlobalID).to receive(:find_many).with(gids, ignore_missing: true)
+      expect(GlobalID).to receive(:find_many).with(gids, {ignore_missing: true})
```
Comment (Benjaminpjacobs) [inline spec/global_id_utils_spec.rb:57]: I assume these are the Ruby 3 updates! 👍🏻

**Review by Benjaminpjacobs — APPROVED** [review]
AFAICT good to go! One question about the gemfile lock change but I don't think it's blocking.

File: Gemfile.lock (hunk 1 above)
Comment (corasaurus-hex) [reply Gemfile.lock:96]: Gah, I fixed this on other branches but missed this one.

File: spec/global_id_utils_spec.rb (hunk 2 above)
Comment (corasaurus-hex) [reply spec/global_id_utils_spec.rb:57]: Yep!

**Pull request merged**